en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-configuration.md

```diff
@@ -0,0 +1,30 @@
+# Setting up ISO8583 Connector 
+
+ISO8583 is an international standard for financial transaction card originated interchange messaging. It is the International Organization for Standardization standard for systems that exchange electronic transactions initiated by cardholders using payment cards.
+
+Typically, whenever we use a credit card, debit card, or ATM card, the data travels from one system to another system. A card-based transaction typically needs to travel between a number of systems.
+ISO 8593 is Financial Transaction card originated, Interchange message specifications adopted by wide segment of Payment Industry (including Acquirers and Issuers).
+
+## Setting up the environment
+
+Before you start configuring the ISO8583 connector, you also need WSO2 MI, and we refer to that location as <PRODUCT_HOME>.
+
+To configure the ISO8583 connector, copy the following client libraries from the given locations to the `<PRODUCT_HOME>/repository/components/lib` directory.
+
+* [jpos-1.9.4.jar](http://mvnrepository.com/artifact/org.jpos/jpos/1.9.4)  
+* [jdom-1.1.3.jar](http://mvnrepository.com/artifact/org.jdom/jdom/1.1.3) 
+* [commons-cli-1.3.1.jar](http://mvnrepository.com/artifact/commons-cli/commons-cli/1.3.1) 
+
+## Configure the test server
+
+For testing purposes, you need to have a test server (basically a Java socket connection that listens on port 5010) to handle ISO8583 requests that come from the connector. You also need to generate responses by changing the relevant response fields, and then send the responses back to the connector. You can test the connector with the sample Java server program that is provided in the following [git location](https://github.com/wso2-docs/CONNECTORS/tree/master/ISO8583/ISO8583TestServer). 
+
+You can include required header information within the header tag in . It supports 2-byte or 4-byte headers. To include header information, you need to convert the 2-byte or 4-byte header into a string using base64 encoding, and then specify the string value within the header tag.
```

Comment:
```suggestion
You can include required header information within the header tag. It supports 2-byte or 4-byte headers. To include header information, you need to convert the 2-byte or 4-byte header into a string using base64 encoding, and then specify the string value within the header tag.
```

Is there some example we can add here or link that we can point to regarding this? It sounds complicated so its better if we can give some more instruction.

Reply:
Simply added reference document for this part.(https://en.wikipedia.org/wiki/ISO_8583).

This section (Configure the test server) not mandatory configuration for setting up the connector. If customer need to set up this connector with their scenarios, they should have their own ISO8583 back-ends. This sample Java server program just only simulates the ISO8583 mock back end behaviors.